apps/chat-e2e/src/tests/promptFoldersSpecialChars.test.ts

```diff
@@ -0,0 +1,225 @@
+import dialTest from '@/src/core/dialFixtures';
+import {
+  ExpectedConstants,
+  ExpectedMessages,
+  MenuOptions,
+} from '@/src/testData';
+import { keys } from '@/src/ui/keyboard';
+import { expect } from '@playwright/test';
+
+dialTest(
+  'Prompt folder: Error message appears if there is a dot is at the end of folder name.\n' +
+    'Prompt folder: allowed special characters.\n' +
+    'Prompt folder: restricted special characters are not entered.\n' +
+    'Prompt folder: restricted special characters are removed if to copy-paste.\n' +
+    'Prompt folder: spaces in the middle of folder name stay.\n' +
+    'Prompt folder: name can not be blank or with spaces only.\n' +
+    'Prompt folder: spaces at the beginning or end of folder name are removed.\n' +
+    'Prompt folder: smiles, hieroglyph, specific letters in name',
+  async ({
+    dialHomePage,
+    promptBar,
+    folderPrompts,
+    folderDropdownMenu,
+    errorToast,
+    setTestIds,
+    page,
+  }) => {
+    setTestIds(
+      'EPMRTC-2975',
+      'EPMRTC-2976',
+      'EPMRTC-2977',
+      'EPMRTC-2978',
+      'EPMRTC-2979',
+      'EPMRTC-2980',
+      'EPMRTC-2981',
+      'EPMRTC-2982',
+    );
+    const folderName = ExpectedConstants.newFolderWithIndexTitle(1);
+    const newNameWithEndDot = `${folderName}.`;
+    const newNameWithSpecialChars = `${folderName} ${ExpectedConstants.allowedSpecialChars}`;
+    const nameWithRestrictedChars = `Folder=,:;{}\\/%&"_name`;
```

Comment:
you may use `ExpectedConstants.restrictedNameChars` to have all restricted symbols in one place

Reply:
Fixed